frontend/src/concepts/areas/const.ts

```diff
@@ -77,5 +77,6 @@ export const SupportedAreasStateMap: SupportedAreasState = {
   [SupportedArea.MODEL_REGISTRY]: {
     featureFlags: ['disableModelRegistry'],
     requiredComponents: [StackComponent.MODEL_REGISTRY],
+    requiredCapabilities: [StackCapability.SERVICE_MESH, StackCapability.SERVICE_MESH_AUTHZ],
```

Comment:
@lucferbux this is further restricting the existing Model Registry page to only work if you have service mesh, is that intended?

Reply:
Assuming the answer is yes since you /lgtm'd, I missed that

Reply:
Yes, that's right, we need both service mesh and authorino to be working, we didn't added it before cause when we started development weren't part of the operator, now it can be safely added.